apps/server/src/modules/group/repo/group.repo.ts

```diff
@@ -65,6 +65,21 @@ export class GroupRepo extends BaseDomainObjectRepo<Group, GroupEntity> {
 		return domainObjects;
 	}
 
+	public async findAvailableByUserAndGroupTypes(user: UserDO, groupTypes?: GroupTypes[]): Promise<Group[]> {
```

Comment:
Why you are using the whole UserDO instread of the id? You only use the id..

Reply:
the already existing method findByUserAndGroupTypes() had already the UserDO given, so i used the same method. I'm not sure what the current state of the architecture says about it 🤷‍♂️